matrix_mqtt_bridge.py

```diff
@@ -51,6 +56,28 @@ async def main() -> None:
         message_type="m.room.message",
         content={"msgtype": "m.text", "body": "[Matrix-MQTT-Bridge]: Ready"},
     )
+    
+    # using MQTT version 5 here, for 3.1.1: MQTTv311, 3.1: MQTTv31
+    # userdata is user defined data of any type, updated by user_data_set()
+    # client_id is the given name of the client
+    global mqtt_client
+    mqtt_client = paho.Client(client_id="Matrix-MQTT-Bridge", userdata=None, protocol=paho.MQTTv31)
```

Comment:
```suggestion
    mqtt_client = paho.Client(callback_api_version=paho.CallbackAPIVersion.VERSION1, client_id="Matrix-MQTT-Bridge", userdata=None, protocol=paho.MQTTv31)
```

Just saw there was a breaking change in the eclipse paho mqtt library. Maybe you can include this fix in your PR? :sweat_smile: 

https://eclipse.dev/paho/files/paho.mqtt.python/html/migrations.html